src/components/learner-credit-management/LearnerCreditAllocationTable.jsx

```diff
@@ -33,35 +39,34 @@ const LearnerCreditAllocationTable = ({
       showFiltersInSidebar={isDesktopTable}
       isLoading={isLoading}
       defaultColumnValues={{ Filter: TableTextFilter }}
+      FilterStatusComponent={FilterStatus}
       columns={[
         {
-          Header: 'Email Address',
-          accessor: 'userEmail',
+          Header: 'Date',
+          accessor: 'enrollmentDate',
           // eslint-disable-next-line react/prop-types, react/no-unstable-nested-components
-          Cell: ({ row }) => <EmailAddressTableCell row={row} enterpriseUUID={enterpriseUUID} />,
+          Cell: ({ row }) => dayjs(row.values.enrollmentDate).format('MMMM DD, YYYY'),
+          disableFilters: true,
         },
         {
-          Header: 'Course Name',
-          accessor: 'courseTitle',
+          Header: 'Description',
+          accessor: getDescriptionAccessor,
+          // eslint-disable-next-line react/prop-types, react/no-unstable-nested-components
+          Cell: ({ row }) => (
+            <>
+              <div>{row.original.courseTitle}</div>
+              <EmailAddressTableCell row={row} enterpriseUUID={enterpriseUUID} />
+            </>
+          ),
+          disableFilters: false,
+          disableSortBy: true,
         },
         {
-          Header: 'Course Price',
+          Header: 'Amount',
           accessor: 'courseListPrice',
           Cell: ({ row }) => `$${row.values.courseListPrice}`,
```

Comment:
Looks like we either will either need to these ESLint rules similar to line 54 above, or define this as a standalone React component (not defined inline).